components/frontend/src/metric/MetricType.js

```diff
@@ -1,4 +1,5 @@
 import { useContext } from 'react';
+import { func, string} from 'prop-types';
```

Comment:
```suggestion
import { func, string } from 'prop-types';
```

Reply:
I want to also add a style checker to check/fix issues like this. That's for another PR.